### ISSUE TITLE
OnAcid-problem with reading S vector(deconvolved trace)

### ISSUE THREAD

**Issue by skasavica** [flatironinstitute/CaImAn#553]
I am running CaImAn OnAcid on Windows 10 operating system. The code works fine and I get the matrix C as number of componets x number of frames. For S matrix I always get a matrix of 2 x 200 (no matter of the number of imaging files), and as I understood it should be the same size as C. Can someone help me with this?

Thank you in advance 

BR

Sergej

- **If you're not reporting an error, type your message below**

**Comment by epnev** [comment]
@skasavica 
Thanks for raising this. Try the following command:
```
S = np.stack([osi.s for osi in cnm.estimates.OASISinstances])
```
I will update the code to do that within the estimates object.

**Comment by epnev** [comment]
fixed with 91e5957

**Comment by skasavica** [comment]
Ok. Thank you very much. It works now.
